crates/pathfinder/src/rpc/v02/method/get_proof.rs

```diff
@@ -0,0 +1,272 @@
+use anyhow::{anyhow, Context};
+use serde::Deserialize;
+
+use crate::core::{BlockId, ContractAddress, StorageAddress};
+use crate::rpc::v02::RpcContext;
+use crate::state::merkle_tree::ProofNode;
+use crate::state::state_tree::{ContractsStateTree, GlobalStateTree};
+use crate::storage::{ContractsStateTable, StarknetBlocksBlockId, StarknetBlocksTable};
+
+#[derive(Deserialize, Debug, PartialEq, Eq)]
+pub struct GetStorageProofInput {
+    pub contract_address: ContractAddress,
+    pub key: StorageAddress,
```

Comment:
```suggestion
    pub key: Vec<StorageAddress>,
```
Should we support multiple storage elements in a single query? Probably :D

Reply:
see 6f23394a8b18726a3f82699090569edabf52727a ! :D